src/stream/consumer-stream.ts

```diff
@@ -44,14 +44,15 @@ async function* genPoll(
 
 
 
-const singleConsumerStream = (config: ClientConfig) => async (
+export const singleConsumerStream = (config: ClientConfig) => async (
   poll: PollMessages,
-  interval: 1000
+  // interval: 1000
```

Comment:
Is it expected to keep the comment?

Reply:
yup it was unused and so raised a lint error, yet i still wanna play around that later probably